### PR TITLE
test(oomstore): add unit test TestPrepareStreamTable

### PR DESCRIPTION
**Pull request by lianxmfor**
this PR is add unit-test `TestPrepareStreamTable`

### CODE REVIEW

File: internal/database/online/dynamodb/stream.go [hunk 1]
```diff
@@ -11,6 +11,12 @@ import (
 )
 
 func (db *DB) PrepareStreamTable(ctx context.Context, opt online.PrepareStreamTableOpt) error {
+	// dynamodb is no such thing as a "column", so if you need to add columns, do not do anything.
```
Comment (YikSanChan) [inline internal/database/online/dynamodb/stream.go:14]: dynamodb has no "column", so we do nothing to "add column". see https://stackoverflow.com/a/25610645/16428442

File: internal/database/online/test_impl/stream.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	}
+}
+
+func TestPrepareStreamTable(t *testing.T, prepareStore PrepareStoreFn, destroyStore DestroyStoreFn) {
+	t.Cleanup(destroyStore)
+
+	ctx, store := prepareStore(t)
+	defer store.Close()
+
+	for _, group := range simpleSteramData.groups {
```
Comment (YikSanChan) [inline internal/database/online/test_impl/stream.go:80]: typo

File: internal/database/online/test_impl/stream.go [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+		t.Run("create steam table", func(t *testing.T) {
+			err := store.PrepareStreamTable(ctx, online.PrepareStreamTableOpt{
+				Entity:  group.Entity,
+				GroupID: group.ID,
+			})
+			assert.NoError(t, err, "create stream table failed: %v", err)
+		})
+	}
+
+	for _, feature := range simpleSteramData.features {
```
Comment (YikSanChan) [inline internal/database/online/test_impl/stream.go:90]: typo

File: internal/database/online/dynamodb/stream.go (hunk 1 above)
Comment (lianxmfor) [reply internal/database/online/dynamodb/stream.go]: fixed

File: internal/database/online/test_impl/stream.go [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+func TestPrepareStreamTable(t *testing.T, prepareStore PrepareStoreFn, destroyStore DestroyStoreFn) {
+	t.Cleanup(destroyStore)
+
+	ctx, store := prepareStore(t)
+	defer store.Close()
+
+	for _, group := range simpleSteramData.groups {
+		t.Run("create steam table", func(t *testing.T) {
```
Comment (YikSanChan) [inline internal/database/online/test_impl/stream.go:81]: typo: steam

**Review by YikSanChan — APPROVED** [review]
lgtm

File: internal/database/online/test_impl/stream.go (hunk 3 above)
Comment (lianxmfor) [reply internal/database/online/test_impl/stream.go]: fixed

File: internal/database/online/test_impl/stream.go (hunk 4 above)
Comment (lianxmfor) [reply internal/database/online/test_impl/stream.go]: fixed

File: internal/database/online/test_impl/stream.go (hunk 2 above)
Comment (lianxmfor) [reply internal/database/online/test_impl/stream.go]: fixed

**Pull request merged**